readme.rst

```diff
@@ -45,16 +46,20 @@ Getting Started
 ---------------
 
 For our first project, we'll do something pretty simple. Make a new
-folder for your project, open a shell there, and type:
+folder for your project, open a shell there, and run ``grunt-init``:
```

Comment:
Is it confusing to say "open a shell" second if you're already in the shell to make the directory?

Reply:
Yes if you're already in a shell; no if you're using the GUI to create a folder. Since the next lines here use a shell to make the new directory, let's remove the shell-opening.